client/store/items.ts

```diff
@@ -19,8 +19,29 @@ export const state = defaultState;
 
 export const getters: GetterTree<ItemsState, ItemsState> = {
   getItem: (state) => (id: string): BaseItemDto | undefined => state.byId[id],
-  getItems: (state) => (ids: string[]): BaseItemDto[] =>
-    map(ids, (id) => state.byId[id])
+  getMissingIds: (state) => (ids: string[]): string[] => {
+    const missingIds = [] as string[];
+
+    ids.forEach((id) => {
+      if (!state.allIds.includes(id)) {
+        missingIds.push(id);
+      }
+    });
+
+    return missingIds;
```

Comment:
```suggestion
    return ids.filter((id) =>  !state.allIds.includes(id))
```

I think this should be better, wdyt?